lib/web/mage/sticky.js

```diff
@@ -41,8 +44,34 @@ define([
 
             if (!isStatic && this.element.is(':visible')) {
                 offset = $(document).scrollTop() - this.parentOffset;
+
+                if (typeof this.options.spacingTop === 'function') {
+                    offset += this.options.spacingTop();
+                } else {
+                    offset += this.options.spacingTop;
+                }
+
                 offset = Math.max(0, Math.min(offset, this.maxOffset));
-                this.element.css('top', offset);
+
+                if (offset &&
+                    this.options.offsetTop &&
+                    !this.element.hasClass(this.options.stickyClass)) {
+
+                    var offsetTop = 0;
+                    if (typeof this.options.offsetTop === 'function') {
+                        offsetTop = this.options.offsetTop();
+                    } else {
+                        offsetTop = this.options.offsetTop;
+                    }
+
+                    if (offset < this.options.offsetTop) {
```

Comment:
This line is a bug; `this.options.offsetTop` may still be a function. You might have means `if (offset < offsetTop)`. The refactor I suggest in my above comments should fix it, though.